csharp/Facebook.CSSLayout/CSSNode.cs

```diff
@@ -0,0 +1,685 @@
+                Native.CSSNodeStyleSetDirection(_cssNode, value);
+            }
+        }
+
+        public CSSFlexDirection FlexDirection
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetFlexDirection(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetFlexDirection(_cssNode, value);
+            }
+        }
+
+        public CSSJustify JustifyContent
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetJustifyContent(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetJustifyContent(_cssNode, value);
+            }
+        }
+
+        public CSSAlign AlignItems
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetAlignItems(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetAlignItems(_cssNode, value);
+            }
+        }
+
+        public CSSAlign AlignSelf
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetAlignSelf(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetAlignSelf(_cssNode, value);
+            }
+        }
+
+        public CSSAlign AlignContent
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetAlignContent(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetAlignContent(_cssNode, value);
+            }
+        }
+
+        public CSSPositionType PositionType
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetPositionType(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetPositionType(_cssNode, value);
+            }
+        }
+
+        public CSSWrap Wrap
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetFlexWrap(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetFlexWrap(_cssNode, value);
+            }
+        }
+
+        public float Flex
+        {
+            get
+            {
+                CheckDisposed();
+                return Native.CSSNodeStyleGetFlex(_cssNode);
+            }
+
+            set
+            {
+                CheckDisposed();
+                Native.CSSNodeStyleSetFlex(_cssNode, value);
+            }
+        }
```

Comment:
All these properties feel like a lot of boilerplate, I wonder if they could be autogenerated?